lib/manageArtifacts.js

```diff
@@ -87,7 +88,7 @@ const storeArtifact = (variables, failIfNotFound) => __awaiter(void 0, void 0, v
         const uploadResponses = yield Promise.all(artifactsUploadPromises);
         for (const variable of variables) {
             core.exportVariable(variable.key, variable.value);
-            core.debug(`Imported ${variable.key}=${variable.value}`);
+            core.debug(`Imported ${variable.key}=${variable.value} and exported as ENV var`);
```

Comment:
You import and export ENV variables at once?

If so, `Imported ${variable.key}=${variable.value} and exported it back as ENV var` might be better

Reply:
Yes exactly, I'm not sure if that's important but it can be usefull.